utils/colors.ts

```diff
@@ -1,4 +1,13 @@
-const colors = {
+interface Colors{
+	"primary":string,
```

Comment:
why are the keys strings?

Reply:
You can resolve this.
I will do it differently as such:
```ts
type ColorName = "primary" | "secondary" | "text" | "invert";
type Colors = { [key: ColorName]: string };

const colors: Record<Theme, Colors> = ...
```

one could say it's an overkill, which is fair. I would say that it's orgenized better this way and when you need to add a theme, you just put another one in the ColorName's union.

Reply:
noted